src/Web/appsettings.json

```diff
@@ -17,12 +17,15 @@
     "DeviceFingerprintingCustomerId": "", //Your Microsoft device fingerprinting customer ID. Typically your Instance ID from the Dynamics 365 Fraud Protection portal dashboard.
     "ApiBaseUrl": "https://api.dfp.microsoft-int.com", //Remove '-int' to call the production API instead.
```

Comment:
shouldn't we change the baseurl and resource below to the dynamics urls?

Reply:
Yes, thanks!